### DISCUSSION TITLE
Fabric Screen API crashing

### DISCUSSION THREAD

**Discussion by LiterallyAidan** [FabricMC/fabric-api#3492] [Player support]
### Minecraft Logs

```txt
https://pastebin.com/5wXytU0S
```

### Description

When I use JEI on modded items the Fabric Screen API crashes and I was wondering if there way a way to fix that

**Comment by Linguardium** [comment]
this the fabric discussion board, you are using forge

**Reply by LiterallyAidan** [reply]
I was told it was the fabric screen api that was crashing. its the better minecraft modpack

**Reply by Linguardium** [reply]
the proper place to get help with the connector mod and related mods (like forgified fabric api) is from the connector dev. their mod is a forge mod not a fabric mod. https://github.com/Sinytra/ForgifiedFabricAPI/issues  https://github.com/Sinytra/Connector/issues
